### PR TITLE
pkg/reportingplugins: remove; move mercury to chainlink-data-streams

### PR DESCRIPTION
**Pull request by jmank88**
https://smartcontract-it.atlassian.net/browse/BCF-2737

Supports:
- https://github.com/smartcontractkit/chainlink-mercury/pull/1
- https://github.com/smartcontractkit/chainlink/pull/11293

### CODE REVIEW

File: pkg/types/provider_mercury.go [hunk 1]
```diff
@@ -1,21 +1,97 @@
...
+	BenchmarkPrice     *big.Int
+}
+
+type MercuryFieldsV3 struct {
+	ValidFromTimestamp uint32
+	Timestamp          uint32
+	NativeFee          *big.Int
+	LinkFee            *big.Int
+	ExpiresAt          uint32
+	BenchmarkPrice     *big.Int
```
Comment (jmank88) [inline pkg/types/provider_mercury.go:78]: It looks like this would be a valid simplification too :thinking: 
```suggestion
	MercuryFieldsV2
```

Comment (vyzaldysanchez) [reply pkg/types/provider_mercury.go:78]: We could go with that... 🤔

**Review by vyzaldysanchez — DISMISSED** [review]

File: pkg/types/provider_mercury.go (hunk 1 above)
Comment (jmank88) [reply pkg/types/provider_mercury.go:96]: I'm not sure it is really helpful and I'm worried about some kind of edge case failing :shrug:

Comment (vyzaldysanchez) [reply pkg/types/provider_mercury.go:96]: That's fair

**Review by vyzaldysanchez — DISMISSED** [review]

**Review by vyzaldysanchez — DISMISSED** [review]

**Review by samsondav — DISMISSED** [review]

**Review by samsondav — APPROVED** [review]

**Pull request merged**